code/game/objects/items/flamethrower.dm

```diff
@@ -25,6 +25,7 @@
 	var/obj/item/weldingtool/weldtool = null
 	var/obj/item/assembly/igniter/igniter = null
 	var/obj/item/tank/internals/plasma/ptank = null
+	var/obj/item/reagent_containers/glass/bker = null
```

Comment:
The description of the flamethrower should probably be updated to say that you can attach a beaker to it.
Also the variable name would likely be better as "beaker" to be more descriptive.